src/WikidataReferencer/WikidataReferencerCommand.php

```diff
@@ -172,6 +173,15 @@ public function initServices() {
 						'P58' => 'author',
 						// Organization
 						'P272' => array( 'creator', 'productionCompany' ),
+						// Content
+						'P364' => 'inLanguage',
+						'P674' => 'character',
+						'P840' => 'contentLocation',
+						//Metadata
+						'P166' => 'award',
+						'P1657' => 'contentRating',
```

Comment:
Matches such as award and contentRating might also need more advanced references (an example can be seen with film genres)